src/integration-test/system-integration-tests.ts

```diff
@@ -4,879 +4,305 @@
-      const url = `${charles}/api/activity?filter=project[${projectId}]`;
-      log(`Fetching activity from ${prettyUrl(url)}`);
-      const ret = await userFetch(url);
-      expect(ret.status).to.equal(200);
-      const json = await ret.json();
-      expect(json.data).to.exist;
-      expect(json.data).to.have.length(1);
-      expect(json.data[0].attributes['activity-type']).to.equal('deployment');
-      expect(json.data[0].attributes.deployment.status).to.equal('success');
-      expect(json.data[0].attributes.project.id).to.equal(String(projectId));
-      expect(json.data[0].attributes.project.name).to.equal(projectName);
-      expect(json.data[0].attributes.commit).to.exist;
-      expect(json.data[0].attributes.branch.name).to.equal('master');
-    });
-
-    it('should be able to edit project', async function () {
-      logTitle('Editing project');
-      this.timeout(1000 * 30);
-      const editProjectPayload = {
-        'data': {
-          'type': 'projects',
-          'id': projectId,
-          'attributes': {
-            'description': 'foo bar bar bar',
-          },
-        },
-      };
-      const ret = await userFetch(`${charles}/api/projects/${projectId}`, {
-        method: 'PATCH',
-        body: JSON.stringify(editProjectPayload),
-      });
-      const json = await ret.json();
-      expect(ret.status).to.equal(200);
-      expect(json.data.id).to.exist;
-      expect(json.data.attributes.description).to.equal(editProjectPayload.data.attributes.description);
-    });
+type TeamType = 'admin' | 'regular' | 'open';
+// const teamTypes: TeamType[] = ['admin', 'regular', 'open'];
+const teamTypes: TeamType[] = ['regular'];
 
-    it.skip('should be able to add comment for deployment', async function () {
-      logTitle('Adding comment');
-      this.timeout(1000 * 10);
-      const addCommentPayload = {
-        data: {
-          type: 'comments',
-          attributes: {
-            email: 'foo@fooman.com',
-            message: 'foo message',
-            name: 'foo',
-            deployment: deploymentId,
-          },
-        },
-      };
-      const ret = await userFetch(`${charles}/api/comments`, {
-        method: 'POST',
-        body: JSON.stringify(addCommentPayload),
+describe('system-integration', () => {
+  for (const teamType of teamTypes) {
+    const projectName = 'regular-project';
+    const notificationIds: number[] = [];
+    let client: CharlesClient;
+
+    describe(`user belonging to '${teamType}' team`, () => {
+
+      const auth0Config = config.auth0[teamType];
+      let oldProjects: any;
+      let deployment: JsonApiEntity | undefined;
+
+      describe('authentication and projects', () => {
+
+        it('should be able to sign in with Auth0', async function () {
+          this.timeout(1000 * 30);
+          const accessToken = await getAccessToken(auth0Config);
+          expect(accessToken).to.exist;
+          client = new CharlesClient(config.charles, accessToken);
```

Comment:
Yep, for sure that is the main drawback in these tests. If some remedies exist, I'd be happy to know.